cypress/e2e/default/learn/challenges/navigation.ts

```diff
@@ -12,6 +12,36 @@ const challenge2 = {
     '/learn/college-algebra-with-python/#build-a-data-graph-explorer-project'
 };
 
+const rwdChallenge = {
+  url: '/learn/2022/responsive-web-design/build-a-personal-portfolio-webpage-project/build-a-personal-portfolio-webpage',
+  nextUrl:
+    'http://localhost:8000/learn/2022/responsive-web-design/#build-a-personal-portfolio-webpage-project'
```

Comment:
Is this supposed to have `http://localhost:8000` in the path?

Reply:
It can, but I would prefer it to work on Gitpod using the `CYPRESS_BASE_URL` env. To move this forward though, we can leave for now.